### PR TITLE
visual: mention `o` key in help

### PR DESCRIPTION
**Pull request by XVilka**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

`o/O` rotates between different print modes - in hexadecimal output

**Test plan**

1. Open any binary
2. Open visual hex mode with `VP`
3. Press `o` and `O` to rotate between different hex representations

CI is green

### CODE REVIEW

File: librz/core/tui/visual.c [hunk 1]
```diff
@@ -298,7 +298,8 @@ static const char *help_msg_visual[] = {
 	"mK/'K", "mark/go to Key (any key)",
 	"n/N", "seek next/prev function/flag/hit (scr.nkey)",
 	"g", "go/seek to given offset (g[g/G]<enter> to seek begin/end of file)",
-	"O", "toggle asm.pseudo and asm.esil",
+	"o/O", "rotate between different formats (next/prev)",
+	"O", "toggle asm.pseudo and asm.esil in disasm mode",
```
Comment (ret2libc) [inline librz/core/tui/visual.c]: Why keeping this line? The same key can't do two different things.

**Review by ret2libc — CHANGES_REQUESTED** [review]

File: librz/core/tui/visual.c (hunk 1 above)
Comment (XVilka) [reply librz/core/tui/visual.c]: The problem - it behaves differently in different visual "submodes"

Comment (ret2libc) [reply librz/core/tui/visual.c]: I think even in disasm mode it doesn't just "toggle asm.pseudo and asm.esil", but it does rotate between several formats (e.g. it also enables asm.bytes, etc.). So I would keep only the first description and remove the second.

Comment (XVilka) [reply librz/core/tui/visual.c]: You are right - I tried and it is indeed rotates between modes.

**Review by ret2libc — APPROVED** [review]

**Pull request merged**